manifests/params.pp

```diff
@@ -99,11 +99,11 @@
       $init_script    = 'puppet:///modules/statsd/statsd-init-rhel'
     }
     'Debian': {
-      $init_location  = '/etc/init/statsd.conf'
+      $init_location  = '/lib/systemd/system/statsd.service'
```

Comment:
Generally maintainer services go into `/lib/systemd/system` and user overrides or apps got into `/etc`. This feels like a system service to me? But otherwise not much in it.